src/shogun/distance/Distance.h

```diff
@@ -335,6 +335,28 @@ class CDistance : public CSGObject
 			return lhs==rhs;
 		}
 
+		/**
```

Comment:
please write more description in the doxygen comments, especially about the parameters...